frontend/packages/dev-console/src/components/topology/componentFactory.ts

```diff
@@ -43,13 +43,15 @@ import {
   TYPE_REVISION_TRAFFIC,
   TYPE_SERVICE_BINDING,
   TYPE_KNATIVE_REVISION,
+  TYPE_TRAFFIC_CONNECTOR_EDGE,
```

Comment:
So consts for other connectors doesnt have an edge suffix, for consistency either add edge suffix in all connectors or remove it from this one.